src/ui/EmojiReactions/index.tsx

```diff
@@ -26,60 +31,44 @@ interface Props {
 }
 
 const EmojiReactions = ({
-  className,
+  className = '',
   userId,
   message,
+  channel,
   emojiContainer,
   memberNicknamesMap,
   spaceFromTrigger = { x: 0, y: 0 },
   isByMe = false,
   toggleReaction,
 }: Props): ReactElement => {
-  const { stringSet } = useContext(LocalizationContext);
-  const emojisMap = getEmojiMapAll(emojiContainer);
+  const { isMobile } = useMediaQueryContext();
   const addReactionRef = useRef(null);
+  const [showEmojiList, setShowEmojiList] = useState(false);
+  const [showEmojisBottomSheet, setShowEmojisBottomSheet] = useState('');
```

Comment:
nit; 
Sounds like `showEmojisBottomSheet` should be a boolean type. Like the above one; `showEmojiList`. It would be better we make it as boolean type or rename it into string like state.